pom.xml

```diff
@@ -197,6 +202,7 @@
     <module>google-cloud-storage</module>
     <module>grpc-google-cloud-storage-v2</module>
     <module>proto-google-cloud-storage-v2</module>
+    <module>proto-google-cloud-storage-control-v2</module>
```

Comment:
@sydney-munro google-cloud-storage-control module is missing. Would you add it?

<img width="313" alt="Screenshot 2024-01-08 at 3 27 35 PM" src="https://github.com/googleapis/java-storage/assets/28604/60a13712-53df-42cd-9ad0-900acf60307e">